.github/workflows/flaky-tests.yaml

```diff
@@ -0,0 +1,81 @@
+name: Flaky Tests Detector
+run-name: Flaky Tests Detector - ${{ github.run_id }} - @${{ github.actor }}
+
+on:
+  pull_request:
+    paths:
+      - '**.go'
+      - 'go.mod'
+      - 'go.sum'
+env:
+  ATTEMPTS: 30
+
+jobs:
+  test:
+    name: Flaky tests detector
+    runs-on: ubuntu-22.04-32core
+
+    permissions:
+      contents: read
+      packages: read
+
+    container:
+      image: ghcr.io/gravitational/teleport-buildbox:teleport12
+      env:
+        TELEPORT_ETCD_TEST: yes
+        TELEPORT_ETCD_TEST_ENDPOINT: https://etcd0:2379
+        TELEPORT_XAUTH_TEST: yes
+        TELEPORT_BPF_TEST: yes
+      options: --cap-add=SYS_ADMIN --privileged
+
+    services:
+      etcd0:
+        image: ghcr.io/gravitational/ci-etcd:3.3.9
+        options: >-
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+          --add-host etcd0:127.0.0.1
+        ports:
+          - 2379:2379
+          - 2380:2380
+          - 3379:3379
+
+    steps:
+      - name: Checkout Teleport
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Prepare workspace
+        uses: ./.github/actions/prepare-workspace
+
+      - name: Mount debugfs
+        run: mount -t debugfs none /sys/kernel/debug/
+
+      - name: Prepare unit tests
+        run: make test-go-prepare
+
+      - name: Run base difftest
+        uses: ./.github/actions/difftest
+        with:
+          flags: --exclude operator/**/*,tool/tsh/**/*,integration/**/*,build.assets/**/*,lib/auth/webauthncli/**/*,lib/auth/touchid/**/*
+          target: test-go-unit
+
+      - name: Run libfido2 difftest
+        uses: ./.github/actions/difftest
+        with:
+          flags: --include lib/auth/webauthncli/**/*
+          target: test-go-libfido2
+
+      - name: Run touch-id difftest
+        uses: ./.github/actions/difftest
+        with:
+          flags: --include lib/auth/touchid/**/*
+          target: test-go-touch-id
+
+      - name: Run tsh difftest
+        uses: ./.github/actions/difftest
+        with:
+          flags: --include tool/tsh/**/*
```

Comment:
I'm curious, why did you decide to split these into different jobs instead of just providing multiple `--include` packages to the difftool command?

Reply:
No. --include supports multiple patterns. Those tests (base, libfido2, touchid, tsh and chaos) were run separately in the original Makefile target, with different flags and tags. I've decided to keep everything as close as possible to the original.